src/data_two.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This structure would be a collection of `Users`. Each `User` is a document with a token and a sub-collection shopping list. The shopping list is a collection of `Items`.

This would represent a user with an empty shopping list (probably a new user):
```
    {
      "user": "ahsdfihwaefwe",
      "shopping_list": []
    },
```

This would represent a user with a shopping list:
```
 {
      "user": "ahsdfihwaefwe",
      "shopping_list": [
        {
          "name": "blah",
          "next_purchase": "soon",
          "last_purchase": "some date string/object"
        },
        {
          "name": "blah",
          "next_purchase": "soon",
          "last_purchase": "some date string/object"
        },
        {
          "name": "blah",
          "next_purchase": "soon",
          "last_purchase": "some date string/object"
        },
        {
          "name": "blah",
          "next_purchase": "soon",
          "last_purchase": "some date string/object"
        }
      ]
```